charts/kubernetes-external-secrets/values.yaml

```diff
@@ -123,3 +123,9 @@ serviceMonitor:
   enabled: false
   interval: "30s"
   namespace:
+
+deploymentInitContainers: {}
+
+emptyDirVolumes: {}
```

Comment:
Can we do more generic and add `extraVolumeMounts` and `extraVolumes` instead ? :)

Reply:
Updated the PR to reflect your suggestion. Thanks for the review 👍